e2e/playwright/package.json

```
@@ -15,6 +15,7 @@
         "@types/pg": "^8.6.6",
         "@typescript-eslint/eslint-plugin": "~5.59.0",
         "@typescript-eslint/parser": "~5.59.0",
+        "mattermost-plugin-e2e-test-utils": "https://github.com/mattermost-community/mattermost-plugin-e2e-test-utils.git#feature/playwright-setup",
```

We'll want to point to a specific tag or commit to lock in deps and source code from the other project

```suggestion
        "mattermost-plugin-e2e-test-utils": "https://github.com/mattermost-community/mattermost-plugin-e2e-test-utils.git#f073874fa4941333c5e540943749aa03f7c32bf2",
```

Taken from latest commit on the PR at this time https://github.com/mattermost-community/mattermost-plugin-e2e-test-utils/commit/f073874fa4941333c5e540943749aa03f7c32bf2